external/js/cothority/src/personhood/ro-pa-sci-instance.ts

```diff
@@ -88,6 +94,25 @@ export default class RoPaSciInstance extends Instance {
         return this.struct.secondPlayer >= 0;
     }
 
+    /**
+     * returns true if there is a CalypsoWrite instance stored.
+     */
+    isCalypso(): boolean {
+        return !this.struct.calypsoWrite.equals(Buffer.alloc(32));
+    }
+
+    /**
+     * ourGame returns if this game belongs to the given coin.
+     * @param coinID
+     */
+    ourGame(coinID: InstanceID): boolean {
+        const player1 = this.struct.firstPlayerAccount;
+        if (player1 !== undefined && !player1.equals(Buffer.alloc(32))) {
+            return player1.equals(coinID);
+        }
+        return !!this.getChoice()[1];
```

Comment:
`return this.getChoice()[1] !== undefined` and possibly change the return type of getChoice to be `Buffer | undefined`